### PR TITLE
Changed Jobboard Model Functions for Search Bar

### PR DESCRIPTION
**Pull request by eldaraharonson**
Changed the jobboard model functions that fetch jobs based on a keyword,
city, or company so that only one search value can be sent to each
function. These functions will be used by the search bar that is being implemented.
Updated the tests accordingly.

### CODE REVIEW

**Review by YarinBou — APPROVED** [review]

File: jobboard/models.py [hunk 1]
```diff
@@ -51,7 +51,11 @@ def get_jobs_by_city_name(cls, city_name):
         """
         Allows students to search for a job based on the city it is located in
         """
-        return set(cls.objects.filter(city=City.objects.get(name=city_name)))
+        try:
```
Comment (guyy13) [inline jobboard/models.py]: I think you can do all of this function in one line like so:
`return set(cls.objects.filter(city__name__iexact=city_name))`
Same goes for the other functions.

Comment (eldaraharonson) [reply jobboard/models.py]: Done

**Review by guyy13 — APPROVED** [review]
lgtm

**Review by yaelerdr — APPROVED** [review]
👍

**Review by benmenachem — APPROVED** [review]

**Review by ifireball — APPROVED** [review]
Approving for now, but given the complexity of the search bar, I suspect you'll need to add additional tests here to test for the tricky edge cases such as wen a city name has a space character in it.
